docs/community/session-managers/agentcore-memory.md

```diff
@@ -41,25 +71,38 @@ session_manager = AgentCoreMemorySessionManager(
     region_name="us-east-1"
 )
 
-# Create agent
+# Create agent with session manager
 agent = Agent(
     system_prompt="You are a helpful assistant. Use all you know about the user to provide helpful responses.",
     session_manager=session_manager,
 )
 
-# Use the agent - conversations are persisted with intelligent retrieval
+# Use the agent - conversations are automatically persisted
 agent("I like sushi with tuna")
-agent("What should I buy for lunch today?")  # Agent remembers preferences
+agent("What should I buy for lunch today?")
```

Comment:
FWIW, this isn't the greatest example because even without a session manager this would work.  Maybe showing a separate agent (agent2) would be a better example as it would show the persistence part